resources/views/contests/voting-multi.blade.php

```diff
@@ -0,0 +1,60 @@
+{{--
+    Copyright 2016 ppy Pty. Ltd.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General P;ublic License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+@extends('contests.base')
+
+@section('contest-content')
+    <div class="contest__description">{!! Markdown::convertToHtml($contest->description_voting) !!}</div>
+    @if ($contest->voting_ends_at !== null && $contest->voting_ends_at->isPast())
+        <div class='contest__voting-notice'>{{trans('contest.voting.over')}}</div>
+    @endif
+    <div class='contest__accordian' id='contests-accordian'>
+        @foreach ($contests as $c)
+            <div class='panel contest__group{{ $loop->first ? ' panel-default' : '' }}'>
```

Comment:
I tried removing the panel (and panel-default) class and it doesn't seem to change anything (apart of missing border radius)...